### PR TITLE
Feature controller specs

### PR DESCRIPTION
**Pull request by rebel216**
Added Controllers Specs.

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @rebel216, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have a professional readme. Kindly make sure to add the updated readme from [here](https://github.com/microverseinc/readme-template#readme).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by sja-thedude** [comment]
Hi @rebel216, 

Your linters are not running properly... Could you check and fix it in the next 5 mins?

**Comment by rebel216** [comment]
> Hi @rebel216,
> 
> Your linters are not running properly... Could you check and fix it in the next 5 mins?

its GitHub issue . please proceed with the project :)
ERROR: [node_modules checker](https://github.com/rebel216/BlogApp-Rails/actions/runs/3587347184/jobs/6037679831)
GitHub Actions has encountered an internal error when running your job.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @rebel216, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have linters setup correctly, kindly delete existing linters and add them from [here](https://github.com/microverseinc/linters-config/tree/master/ror). The linter config files you have are not the right one.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/requests/posts_specs.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 RSpec.describe Post, type: :model do
```
Comment (rloterh) [inline spec/requests/posts_specs.rb]: - [ ] There are no test cases found in the post controller for `index` and `show` actions. Kindly ensure to add appropriate test cases for the post controller, and most importantly you should provide spec test for the `index` and `show` actions in the post controller.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Anomaly, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Linters setup correctly

## Required Changes ♻️
- [ ] Test file for the user controller is not found. Kindly add a test file for user controller and ensure to add spec test for the index and show actions in the user controller.

_Kindly check the other inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Kindly include instructions in your ReadMe on how to run the test cases of your project, under the _Getting Started_ section.
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor. 
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
### Hello @rebel216 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------

**Pull request merged**